packages/fx-core/src/plugins/resource/appstudio/index.ts

```diff
@@ -141,10 +141,16 @@ export class AppStudioPlugin implements Plugin {
    * @param {string} manifestString - the string of manifest.json file
    * @returns {string[]} an array of errors
    */
-  public async validateManifest(ctx: PluginContext): Promise<Result<string[], FxError>> {
+  public async validateManifest(
+    ctx: PluginContext,
+    isLocalDebug = false
+  ): Promise<Result<string[], FxError>> {
     TelemetryUtils.init(ctx);
     TelemetryUtils.sendStartEvent(TelemetryEventName.validateManifest);
-    const validationpluginResult = await this.appStudioPluginImpl.validateManifest(ctx, false);
+    const validationpluginResult = await this.appStudioPluginImpl.validateManifest(
```

Comment:
Maybe we can have code refactor after isConfigUnified() released since no need to distinguish between local and remote anymore